app/src/main/java/fr/neamar/kiss/loader/LoadDocsPojos.java

```diff
@@ -0,0 +1,136 @@
+        if (android.os.Build.VERSION.SDK_INT > Build.VERSION_CODES.GINGERBREAD_MR1) {
+            mimeTypeMap = getMimeTypeMap();
+
+            final String[] DOC_PROJECTION = {
+                    MediaStore.Images.Media._ID,
+                    MediaStore.Images.Media.DATA,
+                    MediaStore.Files.FileColumns.MIME_TYPE,
+                    MediaStore.Files.FileColumns.SIZE,
+                    MediaStore.Images.Media.DATE_ADDED,
+                    MediaStore.Files.FileColumns.TITLE
+            };
+
+            Cursor cursor = null;
+            try {
+                cursor = context.getContentResolver().query(MediaStore.Files.getContentUri("external"),
+                        DOC_PROJECTION,
+                        null,
+                        null,
+                        MediaStore.Files.FileColumns.DATE_ADDED + " DESC");
+
+                if (cursor != null) {
+                    if (cursor.getCount() > 0) {
+                        while (cursor.moveToNext()) {
+                            String path = cursor.getString(cursor.getColumnIndexOrThrow(DATA));
+                            String title = cursor.getString(cursor.getColumnIndexOrThrow(TITLE));
+                            String mimeType = cursor.getString(cursor.getColumnIndexOrThrow(MIME_TYPE));
+                            if (isKnownMimeType(mimeType)) {
+                                docs.add(createPojo(title, path, mimeType, R.drawable.ic_doc));
+                            } else {
+                                String extension = path.substring(path.lastIndexOf(".") + 1, path.length());//.doc .pdf
+                                String extractedMimeType = getMimeType(extension);
+                                if (extractedMimeType != null)
+                                    docs.add(createPojo(title, path, extractedMimeType, R.drawable.ic_doc));
+                            }
+
+                        }
+                    }
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            } finally {
+                if (cursor != null)
+                    cursor.close();
+            }
+        }
+
+        return docs;
+    }
+
+    private String getMimeType(String extension) {
+        if (mimeTypeMap != null) {
+            for (String key : mimeTypeMap.keySet()) {
+                if (key.equals(extension))
+                    return mimeTypeMap.get(extension);
+            }
+        }
+        return null;
+    }
+
+    private boolean isKnownMimeType(String mimeType) {
+
+        return (mimeTypeMap != null && mimeTypeMap.values().contains(mimeType));
+
+    }
+
+    private HashMap<String, String> getMimeTypeMap() {
+        if (context == null)
+            return null;
+        HashMap<String, String> map = new HashMap<>();
+        BufferedReader reader;
+        try {
+
+            InputStream is = context.getAssets().open("general_doc_types.txt");
```

Comment:
ahahha. I concentrated on not sending .DS_Store files and I forgot this file. I will fix them immediately. Thank you for your advice.